tests/rustdoc/attributes-doc-hidden-documented.rs

```diff
@@ -0,0 +1,24 @@
+//@ compile-flags: -Z unstable-options --document-hidden-items
+#![crate_name = "foo"]
+
+//@ has foo/fn.func.html
+//@ has foo/fn.func.html '//pre[@class="rust item-decl"]' '#[doc(hidden)]'
```

Comment:
```suggestion
//@ has - '//pre[@class="rust item-decl"]' '#[doc(hidden)]'
```

and other occurrences

See https://rustc-dev-guide.rust-lang.org/rustdoc-internals/rustdoc-test-suite.html on `-`.

Reply:
Ah yes, I forgot about that option. Thanks!